webapp/app/js/models/job_log_url.js

```diff
@@ -25,7 +25,6 @@ treeherder.factory('ThJobLogUrlModel', [
 
     ThJobLogUrlModel.get_uri = function(){
         var url = thUrl.getProjectUrl("/job-log-url/");
-        $log.log(url);
```

Comment:
Thank you for removing these extra .logs, I've seen the spew in the console on and off and kept on meaning to remove them.